apps/service_providers/llm_service/main.py

```diff
@@ -16,12 +21,18 @@ class LlmService(pydantic.BaseModel):
     def get_raw_client(self) -> SyncAPIClient:
         raise NotImplementedError
 
+    def get_assistant(self, assistant_id: str, as_agent=False):
+        raise NotImplementedError
+
     def get_chat_model(self, llm_model: str, temperature: float) -> BaseChatModel:
         raise NotImplementedError
 
     def transcribe_audio(self, audio: BytesIO) -> str:
         raise NotImplementedError
 
+    def get_callback_handler(self, llm_model: BaseLanguageModel) -> BaseCallbackHandler:
+        return TokenCountingCallbackHandler(llm_model)
```

Comment:
Curious if this should be a list? Maybe there's no need for that currently.

Reply:
yea, it could be multiple